### DISCUSSION TITLE
Get repeat result

### DISCUSSION THREAD

**Discussion by lxygoodjob** [openai/whisper#1730] [Q&A]
When I run two services on a gpu at the same time,  and the GPU usage reaches 100%，there will be repeat text, such as 
"""
Speaker1
0:14:28
对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对对
"""
When I ensure that a service monopolizes a gpu, I will get a very perfect result.
So how can I solve it? I can't ensure that gpu with 80g memory will always be monopolized by whisper.

**Comment by Hkaisense** [comment]
Is your problem solved?

**Comment by toanhuynhnguyen** [comment]
I also faced this issue, I temporarily post-process to remove repeated words.

**Comment by aiyinyuedejustin** [comment]
same, on Macbook M1pro, V3-large. But medium has better English quality and no repetition. V3 large is not as good as expected.
